hooks/environment

```diff
@@ -41,6 +41,9 @@ cat << JSON > "$CREDENTIALS_FILE"
 }
 JSON
 
+GOOGLE_OAUTH_ACCESS_TOKEN="$(gcloud auth print-access-token)"
+
+export GOOGLE_OAUTH_ACCESS_TOKEN
```

Comment:
As the suffix is `_TOKEN`, it should be redacted in the log output ([documentation](https://buildkite.com/docs/pipelines/configure/managing-log-output#redacted-environment-variables)).